src/components/AllergenTable/AllergenTable.tsx

```diff
@@ -8,29 +8,29 @@ interface Props {
 
```

Comment:
This could do with a component test - although because it existed prior to this PR. I'll get one created after you merge.
Must have missed it.